pipenv/vendor/dotenv/main.py

```diff
@@ -102,13 +102,11 @@ def parse_dotenv(dotenv_path):
             if not line or line.startswith('#') or '=' not in line:
                 continue
             k, v = line.split('=', 1)
-
             # Remove any leading and trailing spaces in key, value
-            k, v = k.strip(), v.strip().encode('unicode-escape').decode('ascii')
+            k, v = k.strip(), v.strip()
 
             if len(v) > 0:
                 quoted = v[0] == v[len(v) - 1] in ['"', "'"]
-
                 if quoted:
```

Comment:
It might be better if we put this logic before the decision to encode, since I think it does make sense to escape things by default. Currently this is written such that it does:

1. Encode using unicode-escape, decode back to ascii
2. If the string is quoted, decode using unicode-escape

It might make more sense to just skip the encode/decode step for quoted variables instead